src/app/public/modules/autocomplete/autocomplete-input.directive.ts

```diff
@@ -214,7 +214,15 @@ export class SkyAutocompleteInputDirective implements OnInit, OnDestroy, Control
   public onValidatorChange = () => {};
 
   private setAttributes(element: any): void {
-    this.renderer.setAttribute(element, 'autocomplete', 'off');
+    /**
+     * Modern browsers interpret autocomplete rules differently
+     * and sometimes completely ignore the 'off' value. As a workaround,
+     * 'new-password' will prevent autocomplete in FF 76.0.1, Chrome 81.0, and Edge 81.0.
+     * For preventing autocomplete in Safari, apply directive to a textarea element.
+     * https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion
+     */
+    this.renderer.setAttribute(element, 'autocomplete', 'new-password');
```

Comment:
Thanks for the comments! Would it be too much trouble to add a unit test to confirm the attribute is set correctly, as an added protection from a future engineer removing it?

Reply:
Good point. Done.